sonic-xcvrd/scripts/xcvrd

```diff
@@ -837,6 +837,8 @@ class sfp_state_update_task:
                     #      this is for the vendors who don't implement "system_not_ready/system_becom_ready" logic
                     for key, value in port_dict.iteritems():
                         logical_port_list = platform_sfputil.get_physical_to_logical(int(key))
+                        if logical_port_list is None:
+                            continue
```

Comment:
This change is a good protection to xcvrd, although normal case port_config.ini should include all the FP ports. I would like to suggest to add some syslog here, to avoid cover some configuration error.

Reply:
Good suggestion. Thanks. Added syslog warning message.